.github/workflows/release.yaml

```diff
@@ -38,6 +38,9 @@ jobs:
           base: ${{ steps.branch.outputs.branch }}
           commit-message: "Release update of AutoRevision.txt"
           branch: release/autorevision/${{ steps.branch.outputs.branch }}
+          # Ensure we only add the changed cmake/AutoRevision.txt file in the commit
+          add-paths: |
+            cmake/*.txt
```

Comment:
Why not be even more selective and add `cmake/AutoRevision.txt`?

Reply:
Good question. The docs for the action - https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#add-specific-paths - says we need to "should follow git's pathspec syntax". Both examples have a wildcard, so I followed. In reading the link to the pathspec docs, I don't get a sense you can use the full filename. If you think I can use the full specific file we want to include and ignore everything else I'm good.

Reply:
Pathspecs are basically like shell expansion. SO sent me here: https://css-tricks.com/git-pathspecs-and-how-to-use-them/

The same syntax is e.g. in `.gitignore` and `.gitatribbutes`